.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        required: true
+        type: string
+
+jobs:
+  tests:
+    name: Test Python ${{ inputs.python-version }}
+    runs-on: ubuntu-latest
```

Comment:
This fails when used with python 3.6 because there is no python 3.6 package available for this version of ubuntu (>20.04).